src/components/TokenDetails.tsx

```diff
@@ -22,14 +22,18 @@ import useToken from "hooks/useToken"
 import useWindowSize from "hooks/useWindowSize"
 
 interface Props {
+  contractAddress: string
   id: string
 }
 
-const TokenDetails = ({ id }: Props) => {
+const TokenDetails = ({ contractAddress, id }: Props) => {
   const theme = useTheme()
   const windowSize = useWindowSize()
-  const { loading, error, data } = useToken(id)
+  const { loading, error, data } = useToken(`${contractAddress}-${id}`)
   const token = data?.token
+  const contractConfig = CONTRACT_INFO.filter(
+      x => x.CORE_CONTRACT_ADDRESS.toLowerCase() == contractAddress.toLowerCase()
```

Comment:
nit: I think this should be `===`

Reply:
Addressed with [071be03](https://github.com/ArtBlocks/artblocks-engine-react/pull/10/commits/071be0363e6a4237b3733c943f00446b578e7f0c) in utils/contractInfoHelper.ts